packages/devtools_app/test/test_infra/test_data/memory/heap/heap_graph_fakes.dart

```diff
@@ -173,21 +172,13 @@ class _HeapSnapshotObjectMock implements HeapSnapshotObject {
   final int identityHashCode;
 
   @override
-  HeapSnapshotClass get klass => throw UnimplementedError();
-
-  @override
-  late Uint32List references;
-
-  @override
-  Uint32List get referrers => throw UnimplementedError();
+  Uint32List get references => Uint32List.fromList(_references);
```

Comment:
is there a reason we are making a copy each time?

Reply:
yes, commented